packages/code-du-travail-frontend/pages/_error.js

```diff
@@ -13,17 +13,14 @@ if (typeof window !== "undefined") {
   Sentry.init({ dsn: SENTRY_PUBLIC_DSN, debug: true });
 }
 
-const BrowserException = args => new Error(args);
-
-const notifySentry = err => {
+const notifySentry = statusCode => {
   if (typeof window === "undefined") {
     return;
   }
-  Sentry.configureScope(scope => {
+  Sentry.withScope(scope => {
```

Comment:
we should verify that we don't have duplicated error since I guess _error will be called after an error occur